githooks/pre-commit

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+#
+# Verify proper Java file formatting prior to checkin.
```

Comment:
Should this also ensure that the code compiles?  That seems as important as the formatting.